### DISCUSSION TITLE
redirect_uri

### DISCUSSION THREAD

**Discussion by hannydevelop** [Byron/google-apis-rs#401] [Q&A]
Hi @Byron is there any example of how to assign a custom redirect_uri? I followed this [example](https://github.com/dermesser/yup-oauth2/blob/1c6f9a7d567af5f98c6fb6b57e33a069115dbbf6/examples/custom_flow.rs#L49). However, since it's still using the `HTTPRedirect` for `InstalledFlowReturnMethod`, the token doesn't get to the redirect_uri and authentication isn't completed.

**Comment by Byron** [comment]
For questions related to authentication, please check with the documentation and possibly ask the maintainers of the [`yup-oauth2`](https://crates.io/crates/yup-oauth2) crate. I don't use any of these crates and don't know anything about authflows these days.

**Comment by hannydevelop** [comment]
Hi @Byron I pinged one of the maintainers. While I await their response,  I'll be glad if I can get your input on this:

I'm using the open crate, which works superbly locally. However, when I use this in a production environment (VMS), it tries to open the VM's browser and not that of the user. How best can I open a URL in a user's browser from a production environment?
